Modify switchbot_cloud doc and addd note for webhook support (#36285)
home-assistant/home-assistant.github.io: source/_integrations/switchbot_cloud.markdown

{% warning %}
For IR Appliances, the state is inferred from previous commands in Home Assistant and might not reflect reality if you use other ways to control the device.
{% endwarning %}

## Webhook support

> For vacuums, theirs states are updated from SwitchBot's cloud.

Reviewer: ```suggestion
For vacuums, the states are updated from SwitchBot's cloud.
```

{% warning %}

> Only ONE webhook URL seems to be accepted by the SwitchBot's cloud. So, if you want several applications to be notified, it's not possible, or you'll have to go through a “proxy” which will re-dispatch the message to the other applications.

Reviewer: ```suggestion
Only ONE webhook URL seems to be accepted by the SwitchBot's cloud. So, if you want several applications notified,  you need to use a “proxy” to re-dispatch the message to the other applications.
```
